include/swift/SILOptimizer/Utils/SILInliner.h

```diff
@@ -67,6 +67,17 @@ class SILInliner {
   /// attempts to inline \arg AI and this returns false, an assert will fire.
   static bool canInlineApplySite(FullApplySite apply);
 
+  /// Returns true if inlining \arg apply can result in improperly nested stack
+  /// allocations.
+  ///
+  /// In this case stack nesting must be corrected after inlining with the
+  /// StackNesting utility.
+  static bool needUpdateStackNesting(FullApplySite apply) {
```

Comment:
Usually predicates are written in the third-person singular present, so I think this should be need**s**UpdateStackNesting.